deployment/terraform/efs.go

```diff
@@ -0,0 +1,87 @@
+package terraform
+
+import (
+	"fmt"
+	"strings"
+
+	"github.com/mattermost/mattermost-load-test-ng/deployment/terraform/ssh"
+	"github.com/mattermost/mattermost/server/public/shared/mlog"
+)
+
+const (
+	efsDirectory  = "/mnt/mattermost-data"
+	svcScript     = "/opt/mount.sh"
+	efcServiceDir = "/lib/systemd/system/efs-mount.service"
+	efsService    = `[Unit]
+Description=Script to run after fstab to change ownership of mount directory
+After=local-fs.target
+
+[Service]
+Type=simple
+ExecStart=/bin/bash -c "/opt/mount.sh"
+
+
+[Install]
+WantedBy=multi-user.target
```

Comment:
Nit levels over 9000 (double new line):
```suggestion
[Service]
Type=simple
ExecStart=/bin/bash -c "/opt/mount.sh"

[Install]
WantedBy=multi-user.target
```